src/Factories/JsonErrorFactory.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors\Factories;
+
+use Throwable;
+use WtfPhp\JsonApiErrors\Models\JsonError;
+
+/**
+ * Class JsonErrorFactory
+ * @package WtfPhp\JsonApiErrors\Factories
+ */
+class JsonErrorFactory implements JsonErrorFactoryInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public static function createFromThrowable(Throwable $throwable, $status = 500): JsonError
+    {
+        $jsonError = new JsonError();
+        $jsonError->status = $status;
+        $jsonError->code = $throwable->getCode();
+        $jsonError->title = $throwable->getMessage();
+        $jsonError->detail = $throwable->getTraceAsString();
+        $jsonError->links = [
+            'file' => $throwable->getFile(),
+            'line' => $throwable->getLine()
+        ];
+
+        return $jsonError;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public static function createFromThrowables(array $throwables, $status = 500): array
```

Comment:
I think that's problematic, because every `Throwable` would then have the same status code.

Reply:
Solution: We create an own Exception class, where `id`, `source`, `meta` can also be set, so we are more flexible.
If our own Exception is not used, then we use `500` as a status code.